### PR TITLE
d-mode 2.0.7 (new formula)

### PR DESCRIPTION
**Pull request by rwhogg**
This adds a major mode for the programming language D.

### CODE REVIEW

File: Formula/d-mode.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "739b4623d4a44ab6a61138a60a91f2a591d9bcd318497d09d35820fa3c2bfc6e"
+  head "https://github.com/Emacs-D-Mode-Maintainers/Emacs-D-Mode.git"
+
+  depends_on :emacs => "23"
+  depends_on "cask"
+
+  def install
+    system "make", "compile"
+    elisp.install "d-mode.el", "d-mode.elc"
+    prefix.install "gplv2.txt"
```
Comment (rwhogg) [inline Formula/d-mode.rb]: In my original commit for [groovy-mode](https://github.com/Homebrew/homebrew-emacs/blob/master/Formula/groovy-mode.rb#L25), I put this same file into `doc`. In retrospect, I think that was a mistake; should I make a separate PR to fix that in `groovy-mode`?

Comment (dunn) [inline Formula/d-mode.rb]: Sure, thanks!

**Comment by dunn** [comment]
Travis is acting up, but the only legit failures were popularity so, merged as 84b9b6eaf1cf16450f964cc4ea7eb70c556e28cc.  Thanks again!
